sphinx_js/doclets.py

```diff
@@ -45,8 +48,16 @@ def gather_doclets(app):
                 d)
         except PathTaken as conflict:
             conflicts.append(conflict.segments)
+        except parsimonious.exceptions.ParseError as e:
```

Comment:
And let's import this deeply for consistency with the rest of the project. Also, `e` isn't used, so we can elide it.

Reply:
Import this deeply ?

Reply:
Ah, I guess I should do `from parsimonious.exceptions import ParseError` then.